src/EnergyPlus/AirflowNetwork/include/AirflowNetwork/Elements.hpp

```diff
@@ -446,18 +507,24 @@ namespace AirflowNetwork {
 
         int calculate(bool const LFLAG,           // Initialization flag.If = 1, use laminar relationship
                       Real64 const PDROP,         // Total pressure drop across a component (P1 - P2) [Pa]
-                      int const EP_UNUSED(i),     // Linkage number
+                      int const i,                // Linkage number
+                      const Real64 EP_UNUSED(multiplier),    // Element multiplier
+                      const Real64 EP_UNUSED(control),       // Element control signal
                       const AirProperties &propN, // Node 1 properties
                       const AirProperties &propM, // Node 2 properties
                       std::array<Real64, 2> &F,   // Airflow through the component [kg/s]
                       std::array<Real64, 2> &DF   // Partial derivative:  DF/DP
         );
+
+        virtual ComponentType type()
```

Comment:
This case is a pretty trivial one, but it's nice to mark override functions as `override` to help produce nice compiler warnings if the base class or derived class function accidentally ever changes signature.

Reply:
I'm not a fan of `override` in general, but this time I don't want to add that because in the end the only virtuals will be pure. It's not quite there yet, and there are a couple more tasks to accomplish before that can happen, but it'll get there. This particular method isn't a great example because it's needed for runtime introspection, which I think we should be avoiding and I'm working on removing.